src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1768,7 +1768,7 @@
     <value>{3} '{0}''s type contains the preview type '{1}' and requires opting into preview features. See {2} for more information.</value>
   </data>
   <data name="ModuleInitializerAttributeShouldNotBeUsedInLibrariesDescription" xml:space="preserve">
-    <value>Module initializers are intended to be used by application code to ensure an application's components are initialized before the application code begins executing. If library code declares a 'ModuleInitializer' method, it can interfere with application initialization and also lead to limitations in that application's trimming abilities. Library code should therefore not utilize the 'ModuleInitializer' attribute, but instead expose methods that can be used to initialize any components within the library and allow the application to invoke the method during application initialization.</value>
+    <value>Module initializers are intended to be used by application code to ensure an application's components are initialized before the application code begins executing. If library code declares a method with the 'ModuleInitializerAttribute', it can interfere with application initialization and also lead to limitations in that application's trimming abilities. Library code should therefore not utilize the 'ModuleInitializerAttribute'. Instead of using `ModuleInitializerAttribute` methods, the library should expose methods that can be used to initialize any components within the library and allow the application to invoke the method during application initialization.</value>
```

Comment:
Guess this backtick was not intentional
```suggestion
    <value>Module initializers are intended to be used by application code to ensure an application's components are initialized before the application code begins executing. If library code declares a method with the 'ModuleInitializerAttribute', it can interfere with application initialization and also lead to limitations in that application's trimming abilities. Library code should therefore not utilize the 'ModuleInitializerAttribute'. Instead of using 'ModuleInitializerAttribute' methods, the library should expose methods that can be used to initialize any components within the library and allow the application to invoke the method during application initialization.</value>
```

Reply:
Indeed; thanks!